models/src/main/scala/it/pagopa/interop/authorizationmanagement/model/persistence/events.scala

```diff
@@ -14,13 +14,17 @@ sealed trait Event extends Persistable with ProjectableEvent
 
 final case class KeysAdded(clientId: String, keys: Keys)                                            extends Event
 final case class KeyDeleted(clientId: String, keyId: String, deactivationTimestamp: OffsetDateTime) extends Event
+final case class KeyUpdated(clientId: String, keyId: String, userId: UUID)                          extends Event
```

Comment:
if the purpose of this event is only to migrate relationship ids to user ids I recommend to change its name, because it is too generic and it may be useful in future if we want to actually update a key.
I suggest something like
```suggestion
final case class KeyRelationshipToUserMigrated(clientId: String, keyId: String, userId: UUID)                          extends Event
```